pkgs/development/libraries/gstreamer/core/default.nix

```diff
@@ -41,18 +42,20 @@ stdenv.mkDerivation rec {
     docbook_xsl docbook_xml_dtd_412
   ];
 
-  buildInputs =
-       lib.optionals stdenv.isLinux [ libcap libunwind elfutils ]
-    ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.CoreServices;
+  buildInputs = [ libunwind ]
+    ++ lib.optionals stdenv.isLinux [ libcap elfutils ]
+    ++ lib.optional stdenv.isDarwin CoreServices;
 
   propagatedBuildInputs = [ glib ];
 
   mesonFlags = [
     # Enables all features, so that we know when new dependencies are necessary.
-    "-Dauto_features=enabled"
+    # "-Dauto_features=enabled"
     "-Ddbghelp=disabled" # not needed as we already provide libunwind and libdw, and dbghelp is a fallback to those
     "-Dexamples=disabled" # requires many dependencies and probably not useful for our users
-  ];
+  ]
+    # darwin.libunwind doesn't have pkgconfig definitions so meson doesn't detect it.
+    ++ stdenv.lib.optional stdenv.isDarwin "-DHAVE_UNWIND=1";
```

Comment:
Are you sure this does anything? I do not think meson supports passing compiler flags like this.

Reply:
Yeah, it only worked because of the accidental change above. The problem is that the apple libunwind sources don't contain pkgconfig definitions, I'm not very familiar with meson but we just need a way to to tell it to enable without checking.